### PR TITLE
Decreased downtime and bugs

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
This ever-so-slightly reduces "downtime" events which I felt were obscenely long. I also included some changes that reduce runtime errors, should have no impact on gameplay.

- Cyborg recharge stations - probably the least upgraded machine in existence - receive a flat bonus of +66 per tick, making its base charging rate 66% faster, but pretty inconsequential once upgraded. This clocks in at over a minute, still, to reach full 10000 charge. 
- Ethereals have slightly reduced hunger rate so they're not spamming light bulbs
- Abductors being xeno-immune is not only causing a runtime, but it's not necessarily logical that an alien can't implant another species of alien. They'll surely kill you, anyway.
- Podpeople - yes, they exist! - now mature slightly faster and get some flavor text once being harvested. I hope. I can't test this alone.

## Why It's Good For The Game
I am of the opinion that playing good downtime bad

## Changelog
:cl:
add: Podperson cloning is now a bit more efficient
tweak: T1 Cyborg recharging is no longer abysmally slow.
tweak: Ethereal hunger rate reduced.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/abductors.dm [hunk 1]
```diff
@@ -12,10 +12,10 @@
 	. = ..()
 	var/datum/atom_hud/abductor_hud = GLOB.huds[DATA_HUD_ABDUCTOR]
 	abductor_hud.add_hud_to(C)
-	ADD_TRAIT(C, TRAIT_XENO_IMMUNE, "xeno immune")
+//	ADD_TRAIT(C, TRAIT_XENO_IMMUNE, "xeno immune")
```
Comment (qwertyquerty) [inline code/modules/mob/living/carbon/human/species_types/abductors.dm]: Don't comment out, just remove

File: code/modules/mob/living/carbon/human/species_types/abductors.dm [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
 	abductor_hud.add_hud_to(C)
-	ADD_TRAIT(C, TRAIT_XENO_IMMUNE, "xeno immune")
+//	ADD_TRAIT(C, TRAIT_XENO_IMMUNE, "xeno immune")
 
 /datum/species/abductor/on_species_loss(mob/living/carbon/C)
 	. = ..()
 	var/datum/atom_hud/abductor_hud = GLOB.huds[DATA_HUD_ABDUCTOR]
 	abductor_hud.remove_hud_from(C)
-	REMOVE_TRAIT(C, TRAIT_XENO_IMMUNE, "xeno immune")
+//	REMOVE_TRAIT(C, TRAIT_XENO_IMMUNE, "xeno immune")
```
Comment (qwertyquerty) [inline code/modules/mob/living/carbon/human/species_types/abductors.dm]: Likewise

**Comment by bloons3** [comment]
Does the borg upgraded charger also increase the rate that janitor-borgs have their space lube refilled?
Currently it is extremely slow

**Comment by jlsnow301** [comment]
> Does the borg upgraded charger also increase the rate that janitor-borgs have their space lube refilled?
> Currently it is extremely slow

`var/coeff = recharge_speed * 0.005
R.module.respawn_consumable(R, coeff)`
Indirectly. But I agree with you, so I'm changing it to
recharge_speed*0.025

**Comment by ike709** [comment]
@qwertyquerty

**Comment by ike709** [comment]
@qwertyquerty

**Pull request merged**